### ISSUE TITLE
unable to disable deprecation warning for postcss-preset-env

### ISSUE THREAD

**Issue by redabacha** [csstools/postcss-plugins#328]
### What would you want to propose?

`postcss-custom-properties` now takes an additional option `disableDeprecationNotice` that isn't possible to set when using `postcss-preset-env`. without it, i get a lot of spam in my build output when bundling my app w/ webpack + postcss-loader

### Suggested solution

forward the `disableDeprecationNotice` option to `postcss-custom-properties`

### Validations

- [X] Follow our [Code of Conduct](https://github.com/csstools/postcss-plugins/blob/master/CODE_OF_CONDUCT.md)
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR

**Comment by romainmenke** [comment]
@redabacha Can you share how you tried to set the config?
We did check this :/

**Comment by redabacha** [comment]
oh nvm i'm being dumb. just figured out i wasn't passing the value properly.

**Comment by redabacha** [comment]
yep my config has that now. thanks for the quick response! :100:

**Comment by romainmenke** [comment]
> oh nvm i'm being dumb. just figured out i wasn't passing the value properly.

That is fine :)

There is a lot of config here and missing a bit is expected.
Thank you for opening an issue!
